authentication/views.py

```diff
@@ -100,6 +100,7 @@ class EmailVerificationView(generics.GenericAPIView):
 
     def get(self, request):
         domain = os.environ.get('FRONT_END_LOGIN_URL')
+        print(domain, '======')
```

Comment:
I think this print statement should be removed after use

Reply:
Done bro